src/common.js

```diff
@@ -10,6 +10,9 @@ export const IMPERSONATION_REAL_USER_AUTH_KEY = '_stitch_impers_real_ua';
 export const USER_AUTH_COOKIE_NAME = 'stitch_ua';
 export const DEFAULT_STITCH_SERVER_URL = 'https://stitch.mongodb.com';
 
+// VERSION is substituted with the package.json version number at build time
+export const SDK_VERSION = VERSION;
```

Comment:
This is a good point. If we declare something like `let VERSION = 'unknown'`, I'm pretty sure the plugin will break that by replacing `VERSION` with the version number, making JS complain. If we don't declare it, you're right, then VERSION will be undeclared, making JS complain. I'll think about this more and see what I can do.